src/schema/city/index.ts

```diff
@@ -67,14 +77,9 @@ const CityType = new GraphQLObjectType<any, ResolverContext>({
           // so we have to manually resolve it by lowercasing the value
           // https://github.com/apollographql/graphql-tools/issues/715
           status: args.status.toLowerCase(),
-          // This is to ensure the key never makes its way into the `baseParams`
-          // object if not needed,
-          ...(typeof args.discoverable === "undefined"
-            ? undefined
-            : { discoverable: args.discoverable }),
-          ...(typeof args.discoverable === "undefined"
-            ? { displayable: true }
-            : undefined),
+          include_local_discovery:
+            args.discoverable === true || args.includeStubShows,
+          displayable: args.discoverable === true || args.displayable,
```

Comment:
Just to be sure, this means the `displayable` key will _always_ be included in the params included in the Gravity request; meaning that if _omission_ of the param has any significance in the endpoint then that would no longer work as expected.

Reply:
We don't ever want to show non-displayable shows (where _displayable_ is a longstanding concept having to do with how many artworks/install shots a show has) — unless they are our own Artsy-created stub shows.

So the fact that…

- `displayable=true` is _always_ sent to Gravity, and 
- `include_local_discovery=true` is _sometimes_ sent (in fact always sent for LD use cases)

… sounds 👍  to me.

What I question is why we have `displayable` as an arg to MP at all on L57. I think we can get rid of that, since it would have no effect as @alloy points out, and doesn't support any actual use case.